### PR TITLE
Add readme and compose file

### PR DESCRIPTION
**Pull request by Nsttt**
Funny

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      S3_ACCESS_KEY: "your_s3_access_key"
+      S3_SECRET_KEY: "your_s3_secret_key"
+      INTERVAL: "24h"
+    volumes:
+      - .:/app
+```
+
+## Running the Application with Docker
+
+1. Build the Docker image:
```
Comment (TheDevMinerTV) [inline README.md:58]: The container is also available as `ghcr.io/thedevminertv/postgres_s3_backup`, maybe you could incorporate that?

File: compose.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    build: .
+    environment:
+      URLS: "postgres://user:password@host:port/dbname"
+      S3_ENDPOINT: "your_s3_endpoint"
+      S3_BUCKET: "your_s3_bucket"
+      S3_ACCESS_KEY: "your_s3_access_key"
+      S3_SECRET_KEY: "your_s3_secret_key"
+      INTERVAL: "24h"
+    volumes:
+      - .:/app
```
Comment (TheDevMinerTV) [inline compose.yml]: Not needed

File: compose.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+version: "3.9"
+
+services:
+  pgl_s3_backup:
```
Comment (TheDevMinerTV) [inline compose.yml]:
```suggestion
  pg_s3_backup:
```

**Review by TheDevMinerTV — CHANGES_REQUESTED** [review]
Everything looks fine except for those three things.

File: compose.yml (hunk 2 above)
Comment (Nsttt) [reply compose.yml]: Done

File: README.md (hunk 1 above)
Comment (Nsttt) [reply README.md:58]: Done

**Comment by FinnDore** [comment]
LGTM 👍

**Review by TheDevMinerTV — APPROVED** [review]

**Pull request merged**